BraveRewardsUI/Add Funds/AddFundsViewController.swift

```diff
@@ -39,10 +39,10 @@ class AddFundsViewController: UIViewController {
     }
     
     let map: [(TokenAddressView.TokenKind, String?)] = [
-      (.bitcoin, state.ledger.btcAddress),
-      (.ethereum, state.ledger.ethAddress),
-      (.basicAttentionToken, state.ledger.batAddress),
-      (.litecoin, state.ledger.ltcAddress)
+//      (.bitcoin, state.ledger.btcAddress),
```

Comment:
Because those API were removed and I just needed to get it building… I can either append this PR to remove the entire `AddFundsViewController` or remove it at a future date